docker/test/performance-comparison/compare.sh

```diff
@@ -7,6 +7,10 @@ trap 'kill $(jobs -pr) ||:' EXIT
 stage=${stage:-}
 script_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 
+# https://github.com/jemalloc/jemalloc/wiki/Getting-Started
+export MALLOC_CONF="percpu_arena:disabled"
```

Comment:
```suggestion
```

Looks like this is not used anymore

Reply:
I just want to specify it in all possible ways in hope it's used :)

Reply:
>I just want to specify it in all possible ways 

Got it, but symlink should definitely work

>in hope it's used

Then you can add `confirm_conf:true`, it will definitely show all options

Reply:
Will add on next try. Let's wait for build now, it's too slow to restart and I want to see the result sooner.